packages/fx-core/tests/component/driver/arm/utils.test.ts

```diff
@@ -379,7 +379,7 @@ describe("getDeploymentError", () => {
     deploymentName: "mockDeployName",
     resourceGroupName: "mockRG",
     deploymentStartTime: Date.now(),
-  } as DeployContext;
+  } as any;
```

Comment:
Can you share the reason behind the change?

Reply:
I don't knwo the exact reason why the error emerges when I change LogProvider API. I found the original case is type wrong, I don't know why they can pass the type check.